packages/create-schemas/src/utils.ts

```diff
@@ -0,0 +1,69 @@
+import { isAbsolute, join } from "node:path";
+import { fileURLToPath, pathToFileURL } from "node:url";
+
+function isFileURLAsText(input: string | URL): input is `file://${string}` {
+    return typeof input === "string" && input.startsWith("file://");
+}
+
+function isHttpURLAsText(
```

Comment:
nit: `isHttpUrlAsText` more readable?

Reply:
See other comments. `URL` is uppercased by convention in JS.